app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -85,6 +85,7 @@ import mozilla.components.ui.tabcounter.TabCounterMenu
 import org.mozilla.fenix.BrowserDirection
 import org.mozilla.fenix.HomeActivity
 import org.mozilla.fenix.R
+import org.mozilla.fenix.addons.runIfFragmentIsAttached
```

Comment:
nit: This is in the addons package and I guess everyone started using it now. We should move this out in a follow-up.

Reply:
Opened https://github.com/mozilla-mobile/fenix/issues/17303